### ISSUE TITLE
support spring boot 2.2

### ISSUE THREAD

**Issue by jangalinski** [toolisticon/springboot-swagger-starter#15]
somehow the yaml parsing must have changed, with sb 2.0 I get

```

java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'swagger.api-info' to io.toolisticon.springboot.swagger.ApiInfoProperty
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.lang.String.charAt(String.java:658)

```

**Comment by jangalinski** [comment]
failing build https://travis-ci.org/toolisticon/springboot-swagger-starter/builds/349190448

see branch spring-boot-2

**Comment by jangalinski** [comment]
fixed with e7f00d778a7560e4548d525214ec832523765616
